app/views/steps/shared/_relationship_step.html.erb

```diff
@@ -6,7 +6,7 @@
 
 <%= step_form form_object do |f| %>
   <%=
-    f.radio_button_fieldset :relation do |fieldset|
+    f.radio_button_fieldset :relation, { legend_options: { class: 'visually-hidden', text: t('.heading', person_name: form_object.record.person.full_name, child_name:  form_object.record.minor.full_name) } } do |fieldset|
```

Comment:
I think in cases where we need to pass arguments to `t()` it might be better and cleaner to declare a variable once, and reuse it in the header and in the legend.

For example, in the top of the page:

```ruby
heading = t('.heading', person_name: form_object.record.person.full_name, child_name: form_object.record.minor.full_name)
```

And then we use `heading` variable in the `<h1>` and in the `legend_options `

Reply:
Good call! I have added this here and into other places where the heading was more than just '.heading'